src/Controls/src/Core/Handlers/Shell/ShellContentHandler.Windows.cs

```diff
@@ -16,14 +16,6 @@ public ShellContentHandler() : base(Mapper, CommandMapper)
 		{
 		}
 
-		internal static void MapTitle(ShellContentHandler handler, ShellContent item)
```

Comment:
The mapper was not invoking the method never?

Reply:
Previously, the MapTitle method was only called when changing the title within the same ShellContent page, preventing title changes between different ShellContent pages. Since other platforms support this behavior, I removed MapTitle and applied the fix to the ShellSection. With this change, the title now updates correctly during the initial load, at runtime when adding ShellContent, and when changing title between ShellContent pages, ensuring consistency across all platforms

Reply:
One difference between Windows and the other platforms is that Windows is the only platform that has the handlers correctly built against shell. So, I think we can use that to our advantage a bit here. 

Like we could move all this code to an extention method and then on the shellcontent if the Title property changes we just call that extention method

Reply:
@PureWeen, Should we add the extension only for Windows ShellContent title changes, or should we consider applying this extension to other platforms as well

Reply:
I think this will only currently apply to windows because of how the handlers are setup on Windows. 

If it's straight forward to apply it to other platforms then you can try that, but, if it's not straight forward than just windows is fine

Reply:
@PureWeen, Since applying this to other platforms is not straightforward, I have moved the Windows-specific changes to an extension method for now